packages/rexray/extra/dcos-rexray.service

```diff
@@ -8,4 +8,8 @@ RestartSec=15
 LimitNOFILE=16384
 EnvironmentFile=/opt/mesosphere/environment
 Environment=REXRAY_HOME=/
+# REX-Ray fails to clean up stale lock files by itself.
+ExecStartPre=/bin/rm -f /var/run/libstorage/lsx.lock
+# Allow REX-Ray configuration to be changed during upgrade.
```

Comment:
this comment implies that we've tested a migration scenario (from old dc/os to new dc/os w/ an upgraded rexray)?